File: src/client/ts/models/message.ts
import { NetworkCommands } from "./network-commands"

export class Message {
    public NetworkCommand: NetworkCommands;
    public Author: number;
    public Content: string;

    constructor(networkCpmmand: NetworkCommands, author: number, content: string) {
        this.NetworkCommand = networkCpmmand;
        this.Author = author;
        this.Content = content;
    }
}

File: src/server/Message.cs
namespace EchoApp
{
    class Message
    {
        public readonly NetworkCommands NetworkCommand;
        public readonly int Author;
        public readonly string Content;
        
        public Message(NetworkCommands networkCommand, int author, string content)
        {
            NetworkCommand = networkCommand;
            Author = author;
            Content = content;
        }
    }
}